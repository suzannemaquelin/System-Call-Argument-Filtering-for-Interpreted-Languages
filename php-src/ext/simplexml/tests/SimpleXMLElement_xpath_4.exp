Notice: Undefined variable: x in %s on line %d

Warning: simplexml_load_string(): Invalid options in %s on line %d
bool(false)