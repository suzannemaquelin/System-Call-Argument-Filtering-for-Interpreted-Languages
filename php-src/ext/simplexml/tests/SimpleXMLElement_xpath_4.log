
---- EXPECTED OUTPUT
Notice: Undefined variable: x in %s on line %d

Warning: simplexml_load_string(): Invalid options in %s on line %d
bool(false)
---- ACTUAL OUTPUT
Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory

Notice: Undefined variable: x in /root/php-src/ext/simplexml/tests/SimpleXMLElement_xpath_4.php on line 2

Warning: simplexml_load_string(): Invalid options in /root/php-src/ext/simplexml/tests/SimpleXMLElement_xpath_4.php on line 2
bool(false)
---- FAILED
