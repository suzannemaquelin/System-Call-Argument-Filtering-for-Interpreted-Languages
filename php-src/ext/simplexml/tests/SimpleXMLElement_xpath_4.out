Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory

Notice: Undefined variable: x in /root/php-src/ext/simplexml/tests/SimpleXMLElement_xpath_4.php on line 2

Warning: simplexml_load_string(): Invalid options in /root/php-src/ext/simplexml/tests/SimpleXMLElement_xpath_4.php on line 2
bool(false)